fdbserver/RestoreLoader.actor.cpp

```diff
@@ -518,27 +531,39 @@ ACTOR static Future<Void> _parseRangeFileToMutationsOnLoader(VersionedMutationsM
 // Parse data blocks in a log file into a vector of <string, string> pairs. Each pair.second contains the mutations at a
 // version encoded in pair.first Step 1: decodeLogFileBlock into <string, string> pairs Step 2: Concatenate the
 // pair.second of pairs with the same pair.first.
-ACTOR static Future<Void> _parseLogFileToMutationsOnLoader(
-    std::map<Standalone<StringRef>, Standalone<StringRef>>* pMutationMap,
-    std::map<Standalone<StringRef>, uint32_t>* pMutationPartMap, Reference<IBackupContainer> bc, Version version,
-    std::string fileName, int64_t readOffset, int64_t readLen, KeyRange restoreRange, Key addPrefix, Key removePrefix,
-    Key mutationLogPrefix) {
+ACTOR static Future<Void> _parseLogFileToMutationsOnLoader(NotifiedVersion* pProcessedFileOffset,
+                                                           SerializedMutationListMap* pMutationMap,
+                                                           SerializedMutationPartMap* pMutationPartMap,
```

Comment:
There's not a lot of precedent in our code for Hungarian notation, but I don't have strong opinions.